scripts/airgap/image-list.txt

```diff
@@ -1,5 +1,5 @@
-docker.io/rancher/klipper-helm:v0.9.3-build20241008
-docker.io/rancher/klipper-lb:v0.4.9
+docker.io/rancher/klipper-helm:v0.9.4-build20250113
```

Comment:
To change the klipper-helm image, you need to bump the image in helm-controller, tag a new release, and then bump that here in go.mod.